docs/blog/2023-Q2-community-board.rst

```diff
@@ -0,0 +1,58 @@
+.. post:: May 16, 2023
+   :tags: community board, news, report
+   :author: Mikey Ariel
+
+Write the Docs Community Board Quarterly Update Q2-2023
+=========================================================
+
+Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board.
+
+Last quarter we published our first `quarterly update blog post <https://www.writethedocs.org/blog/2023-Q1-community-board/>`_, where we introduced the team and gave an overview of how the board will aim to support and evolve the community. 
+
+Q2 quarterly meeting highlights
+-------------------------------
+
+Our second meeting focused on further refining our meta-work, such as where the community board tracks work and how we communicate with each other and with the community. 
+
+Historically, WTD used a Kanban board to track ideas for community initiatives and cross-functional tasks, such as infrastructure maintenance and meetup support. 
+Between our Q1 meeting and the Q2 meeting, we have spent some time reviewing and organizing this Kanban board, with the intention of landing on a "clean backlog" that will help us prioritize work and help the team leads to track their tasks, share resources, and ask for help. 
+
+During the meeting we looked at the main areas of the community that had active tasks in the backlog, and agreed to take the information back to our respective teams, as well as consider them as team leads for further triage in the upcoming months. 
+
+Another topic of discussion was the the #bipoc Slack channel and BIPOC stories in the newsletter. 
+Recently, it has become apparent that the community engagement was lower in those spaces, so in order to prevent burnout of the coordinator we recommended to take a breather from actively sourcing content. 
+
+Our community has always voted with its time and energy, and we believe that when the need arises again for content and discussions in those spaces, the community will make sure that we are aware of it and we will be happy to support it. 
+The announcement and thank-you to the coordinator, Heather Zoppetti, was published in the `May 2023 newsletter <https://www.writethedocs.org/blog/newsletter-may-2023/>`_.
+
+As we look forward to our Q3 meeting, we will continue to refine the backlog to determine what are the most valuable efforts for the community in the short and long term. 
+We also hope to look at how the first in-person Portland conference since the before-times went, and what can we learn as we evolve our events in the upcoming months and years.
+
+What is the community board
+---------------------------
+
+The `Community Board <https://www.writethedocs.org/team/#community-board>`_ exists to curate the overall strategic direction of the community.
```

Comment:
I believe it's direction of the efforts of the folks working on the community, not the community itself?